cmd/params/params.go

```diff
@@ -470,16 +466,16 @@ func loadSanitizeParams(v *viper.Viper) (SanitizeParams, error) {
 		)
 	}
 
-	params.HideFileNames = hideFileNamesPatterns
-
-	return params, nil
+	return SanitizeParams{
+		HideBranchNames:     hideBranchNamesPatterns,
+		HideFileNames:       hideFileNamesPatterns,
+		HideProjectFolder:   vipertools.FirstNonEmptyBool(v, "relative-paths", "settings.relative_paths"),
+		HideProjectNames:    hideProjectNamesPatterns,
+		OverrideProjectPath: vipertools.GetString(v, "project-folder"),
+	}, nil
 }
 
 func loadProjectParams(v *viper.Viper) (ProjectParams, error) {
-	if v == nil {
```

Comment:
Just want to make sure this won't cause a panic if for some reason `v` is `null`?

Reply:
Null check is done at the `Load` func.